src/modules/fancyzones/FancyZonesLib/DraggingState.cpp

```diff
@@ -6,7 +6,8 @@
 
 DraggingState::DraggingState(const std::function<void()>& keyUpdateCallback) :
     m_mouseState(false),
```

Comment:
Minor update: let's clarify that it's a secondary mouse state to make it more readable and clean.
`m_mouseState` -> `m_secondaryMouseState`